services/docker-compose-ops.yml

```diff
@@ -82,7 +82,8 @@ services:
         scheduled_maintenance:${REDIS_HOST}:${REDIS_PORT}:3,
         user_notifications:${REDIS_HOST}:${REDIS_PORT}:4,
         announcements:${REDIS_HOST}:${REDIS_PORT}:5,
-        distributed_identifiers:${REDIS_HOST}:${REDIS_PORT}:6
+        distributed_identifiers:${REDIS_HOST}:${REDIS_PORT}:6,
+        distributed_caches:${REDIS_HOST}:${REDIS_PORT}:7
```

Comment:
@YuryHrytsuk I'm not sure if the relative env var needs to be updated somewhere in OPS to reflect these changes

Reply:
It does!

Reply:
The new DB needs to be added here in this line https://github.com/ITISFoundation/osparc-ops-environments/blob/9c79757dfd9ff13ddbcb277cc44ddadab770e5e7/services/redis-commander/docker-compose.yml#L12

Reply:
thanks @mrnicegyu11 I have a PR in place now. https://github.com/ITISFoundation/osparc-ops-environments/pull/501
Also please let me know when this new PR should be merged in.